src/AdminConsole/Components/Shared/ApexCharts/Models/Yaxis.cs

```diff
@@ -0,0 +1,6 @@
+namespace Passwordless.AdminConsole.Components.Shared.ApexCharts.Models;
+
+public class Yaxis
```

Comment:
Yaxis should probably be `YAxis`

Reply:
[No, it would mess with the serialization: https://apexcharts.com/docs/formatting-axes-labels/](https://apexcharts.com/docs/formatting-axes-labels/)